### DISCUSSION TITLE
EJBCA Support for ARM Architecture on Asahi Linux

### DISCUSSION THREAD

**Discussion by hl94vul3h6** [Keyfactor/ejbca-ce#495] [Q&A]
Hi EJBCA Team,

I am interested in installing EJBCA natively on my M1 Max MacBook Pro (not via Docker). I understand that EJBCA officially supports Windows and Linux. However, given that my MacBook runs on an ARM architecture, I am considering using Asahi Linux to install a native Linux environment.

Could you please confirm if EJBCA is compatible with the ARM architecture when running on a Linux distribution like Asahi Linux? Are there any known issues or limitations I should be aware of?

Thank you for your guidance and support.

Best regards,
Watson

**Comment by primetomas** [comment]
EJBCA software is a pure Java application. It will work everywhere that you have OpenJDK, and this is available both on Mac and ARM Linux. EJBCA software you can run directly on a Mac as well if you like.

It is only the docker container that is not built for ARM, but that is related to the Linux base system in the container, EJBCA itself is OS/CPU independent.
